Error: Cannot find module '../src/pinger'

Error: Cannot find module '../src/pinger'

<img width="798" alt="截圖 2021-04-11 下午1 03 26" src="https://user-images.githubusercontent.com/46012524/114293162-8a559a00-9ac6-11eb-9729-15b361e597c9.png">


## To fix
change 
```ts
import { Pinger } from '../src/pinger';
```
to
```ts
import { Pinger } from './pinger';
```

--- a/src/integ.default.ts
+++ b/src/integ.default.ts
@@ -1,5 +1,5 @@
 import * as cdk from '@aws-cdk/core';
-import { Pinger } from '../src/pinger';
+import { Pinger } from './pinger';
 
 export class IntegTesting {
   readonly stack: cdk.Stack[];

